osxkeychain.go

```diff
@@ -57,6 +63,11 @@ func (attributes *GenericPasswordAttributes) CheckValidity() error {
 	if err := check32Bit("Password", attributes.Password); err != nil {
 		return err
 	}
+	for _, trustedApplication := range attributes.TrustedApplications {
+		if err := check32BitUTF8("TrustedApplications", trustedApplication); err != nil {
```

Comment:
can you pass in `fmt.Sprintf("TrustedApplications[%d]", i)` for the first arg to check32BitUTF8?